### ISSUE TITLE
RG03 System shall implement a responsive UI that adapts to mobile platforms

### ISSUE THREAD

**Issue by gubuntu** [kartoza/ckanext-dalrrd-emc-dcpr#261]
Target device form factors shall be:
- Consumer grade desktop computer workstations and laptops
- Entry level mobile phones and tablets

**Comment by gubuntu** [comment]
@Fanevanjanahary you need to show screenshots and provide comments on all issues when you consider them done

**Comment by Jeremy-Prior** [comment]
Consumer grade desktop computer workstations and laptops:
- The UI works on laptops I have tested it on.

Entry level mobile phones and tablets:
- The UI adapts to mobile for most pages. The screenshot below is of the home page from a mobile device:
![Screenshot_20221110-110118_Firefox](https://user-images.githubusercontent.com/100694811/201055868-277b9355-59ad-4546-b532-78266be495bb.jpg)

- The text scales for the Help page but the images do not, could this be because of the image sizes when they are uploaded? I tried to log in to check the unlisted documents but was unable to login, see point below. The screenshot below is zoomed out view of the Help page to fit the images into frame:
![Screenshot_20221110-110245_Firefox](https://user-images.githubusercontent.com/100694811/201057610-9e2a6219-57b5-42d7-910f-68d522167b10.jpg)

- The login page scales but clicking on "Login" doesn't appear to do anything and a user is unable to deselect the "Remember Me" option. The screen shot is just showing the login page:
![Screenshot_20221110-110056_Firefox](https://user-images.githubusercontent.com/100694811/201058132-6b1975cf-4e64-4f0d-a463-db114d8fbcdc.jpg)
